deny.toml

```diff
@@ -35,22 +17,13 @@ targets = [
 db-path = "~/.cargo/advisory-db"
 # The url(s) of the advisory databases to use
 db-urls = ["https://github.com/rustsec/advisory-db"]
-# The lint level for security vulnerabilities
-vulnerability = "deny"
```

Comment:
Is it safe to remove it? Are those things were configured same as default?

Reply:
Deny is the default behavior, that's why it threw errors, so the handling for some reason were not to update, but that's actually the same without quirks